packages/react-data-grid/src/Canvas.js

```diff
@@ -218,9 +218,13 @@ class Canvas extends React.PureComponent {
   };
 
   setScrollLeft = (scrollLeft) => {
+    if (this.interactionMasks && this.interactionMasks.setScrollLeft) {
```

Comment:
Using the same same approach to fix mask's position

Reply:
since you are touching this file, maybe we can also add some simple doc above the function? like

```
/**
 * what does scrollLeft do
 * @param scrollLeft number
 */
```